src/fragments/lib/storage/js/get-properties.mdx

```diff
@@ -0,0 +1,25 @@
+Get the properties of the file using the `getProperties` method.
```

Comment:
can we change this to have the same wording as the Flutter one?

You can get file properties and metadata without downloading the file using Storage.getProperties API